src/main/java/net/dv8tion/jda/core/managers/PermOverrideManager.java

```diff
@@ -202,9 +202,18 @@ public PermOverrideManager grant(long permissions)
         if (permissions == 0)
             return this;
         setupValues();
-        this.allowed |= permissions;
-        this.denied &= ~permissions;
-        this.set |= ALLOWED;
+        if ((allowed & permissions) == 0)
+        {
+            this.allowed |= permissions;
+            this.set |= ALLOWED;
+        }
```

Comment:
So what this code now does: if **none** of the granted permissions are currently granted we set them.
This is really not how this is supposed to go, we want to update if **any** of the permissions are newly granted.

My suggestion was to only change the `this.set |= ALLOWED`.